docs/ocs-on-a-disconnected-environment.md

```diff
@@ -53,129 +53,174 @@ you should get something similar to this:
+  registry.redhat.io/openshift4/ose-csi-driver-registrar@sha256:797ff087ed64e38d0de12875f58ea395ec222bf67cad59a6abb6c131e12bfb97=my.mirror-registry:5000/openshift4/ose-csi-driver-registrar
+  registry.redhat.io/openshift4/ose-csi-external-attacher@sha256:72fcb2c114b7a4b0222802d4fe82ada7620162b1120462942c8a9d8e66230677=my.mirror-registry:5000/openshift4/ose-csi-external-attacher
+  registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:203417dc512f0e3583be06db7f93a3729fef08d8e939ec1fc828ef914d1d8995=my.mirror-registry:5000/openshift4/ose-csi-external-provisioner-rhel7
+  registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:ad5dc22e6115adc0d875f6d2eb44b2ba594d07330a600e67bf3de49e02cab5b0=my.mirror-registry:5000/rhscl/mongodb-36-rhel7
+  quay.io/noobaa/pause@sha256:b31bfb4d0213f254d361e0079deaaebefa4f82ba7aa76ef82e90b4935ad5b105=my.mirror-registry:5000/noobaa/pause
+  docker.io/kubernetes/pause:latest=my.mirror-registry:5000/kubernetes/pause
+  ```
+
+  mapping.txt for OCS-4.3.0:
+  ```
+  registry.redhat.io/ocs4/cephcsi-rhel8@sha256:9aa61eec1d23169521f83b4664115d9aa69710d69d7f2f4a939ec753d15c0400=my.mirror-registry:5000/ocs4/cephcsi-rhel8
+  registry.redhat.io/ocs4/mcg-core-rhel8@sha256:74d03cc253774bd817750ba0fe3cfec125282f8549343067610799f2eee5ea80=my.mirror-registry:5000/ocs4/mcg-core-rhel8
+  registry.redhat.io/rhceph/rhceph-4-rhel8@sha256:9e521d33c1b3c7f5899a8a5f36eee423b8003827b7d12d780a58a701d0a64f0d=my.mirror-registry:5000/rhceph/rhceph-4-rhel8
+  registry.redhat.io/openshift4/ose-csi-driver-registrar@sha256:b40c8e8392e432f49c4abf6b5248fb6a349ab9dff49c7f9467df3fc974937a6d=my.mirror-registry:5000/openshift4/ose-csi-driver-registrar
+  registry.redhat.io/openshift4/ose-csi-external-attacher@sha256:faeb96365fafba1255ca3b9ad7e3f6ddd8791588b9ff283a7acffd7d5c89fe09=my.mirror-registry:5000/openshift4/ose-csi-external-attacher
+  registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:0570eacd3cb8b5d60fd3ecf4c80860280725c81d83bc2a04326d3393c6dcb2be=my.mirror-registry:5000/openshift4/ose-csi-external-provisioner-rhel7
+  registry.redhat.io/openshift4/ose-csi-external-resizer-rhel7@sha256:e7302652fe3f698f8211742d08b2dcea9d77925de458eb30c20789e12ee7ae33=my.mirror-registry:5000/openshift4/ose-csi-external-resizer-rhel7
+  registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:254c9046eaf61c451b40bac3f897c40bc59f187e5313b9799fbcacf17398a191=my.mirror-registry:5000/rhscl/mongodb-36-rhel7
+  quay.io/noobaa/pause@sha256:b31bfb4d0213f254d361e0079deaaebefa4f82ba7aa76ef82e90b4935ad5b105=my.mirror-registry:5000/noobaa/pause
+  docker.io/kubernetes/pause:latest=my.mirror-registry:5000/kubernetes/pause
+  ```
+
+- in OCS-4.4 the relatedImages do appear in the CSV, but we still need to fix the missing pause image for lib-bucket-provisioner.  
+Also the `ose-csi-external-resizer` image is missing in the relatedImage (Although it is not yet used by OCS we'll mirror it anyway)
+
+  mapping.txt for OCS-4.4.0:
```

Comment:
I believe the @sha changed for most OCS 4.4 images, here is list I have (including tag v4.4 for must-gather image)

```
registry.redhat.io/ocs4/cephcsi-rhel8@sha256:6b0b8fe1b7c809381f93e874c8b89b3cbb0ad7f183c8bc06c441efe48cf0a746=<your_registry>/ocs4/cephcsi-rhel8
registry.redhat.io/ocs4/mcg-core-rhel8@sha256:527b648dbb9d2754eaecd24d6d9eaf36c4434994ad59d65013d18e06ae400c23=<your_registry>/ocs4/mcg-core-rhel8
registry.redhat.io/ocs4/mcg-rhel8-operator@sha256:701eae453b29646182533719fa312731bebf1fb53414762894324813b0d7a17a=<your_registry>/ocs4/mcg-rhel8-operator
registry.redhat.io/ocs4/ocs-rhel8-operator@sha256:78b97049b194ebf4f72e29ac83b0d4f8aaa5659970691ff459bf19cfd661e93a=<your_registry>/ocs4/ocs-rhel8-operator
registry.redhat.io/ocs4/rook-ceph-rhel8-operator@sha256:708c5e3b3d075235b8031618e2342d4d8df383d191e197fb81a7f7416494eb14=<your_registry>/ocs4/rook-ceph-rhel8-operator
registry.redhat.io/openshift4/ose-csi-driver-registrar@sha256:85dedabd1e0dc51fda054fc51eee571d274f3ae3a088acf768c18ad73d60406c=<your_registry>/openshift4/ose-csi-driver-registrar
registry.redhat.io/openshift4/ose-csi-external-attacher@sha256:19a6384b7a17e5781358a5264a8f9d137e2a2118c903ae8010c3c216e0f8e588=<your_registry>/openshift4/ose-csi-external-attacher
registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:535379c05aa75405254bf0139dfad43dd8568ae6de37650746c8c961e473dcf2=<your_registry>/openshift4/ose-csi-external-provisioner-rhel7
registry.redhat.io/openshift4/ose-csi-external-resizer-rhel7@sha256:e7302652fe3f698f8211742d08b2dcea9d77925de458eb30c20789e12ee7ae33=<your_registry>/openshift4/ose-csi-external-resizer-rhel7
registry.redhat.io/rhceph/rhceph-4-rhel8@sha256:9e521d33c1b3c7f5899a8a5f36eee423b8003827b7d12d780a58a701d0a64f0d=<your_registry>/rhceph/rhceph-4-rhel8
registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:fa0d0c43f79410d7b2edeb78329e6bf5c9ca5ced141b12341eb34a4042a1dc0a=<your_registry>/rhscl/mongodb-36-rhel7
registry.redhat.io/ocs4/ocs-must-gather-rhel8@sha256:823e0fb90bb272997746eb4923463cef597cc74818cd9050f791b64df4f2c9b2=<your_registry>/ocs4/ocs-must-gather-rhel8
quay.io/noobaa/lib-bucket-catalog@sha256:b9c9431735cf34017b4ecb2b334c3956b2a2322ce31ac88b29b1e4faf6c7fe7d=<your_registry>/noobaa/lib-bucket-catalog
quay.io/noobaa/pause@sha256:b31bfb4d0213f254d361e0079deaaebefa4f82ba7aa76ef82e90b4935ad5b105=<your_registry>/noobaa/pause
```

Reply:
* in 4.4 most of the images are already included in the `relatedImages`. the `ose-csi-external-resizer-rhel7` was left out by mistake, and it should be fixed in 4.5
* regarding the must-gather, If I remember correctly it is not included in the related images. it should be fixed for 4.5 and also in the docs here